packages/govern-console/src/containers/HomePage/HomePage.tsx

```diff
@@ -13,13 +13,17 @@ import CreateDao from 'containers/CreateDao/CreateDao';
 import { trackPage } from 'services/analytics';
 import { useLocation } from 'react-router-dom';
 import { useEffect } from 'react';
+import scrollToTop from '../../utils/scrollToId';
```

Comment:
@Rekard0 could you import this without relative path ? the same kind of way as `services/analytics` is imported above.